Rasa_Bot/requirements.txt

```diff
@@ -1,4 +1,4 @@
 -r actions/requirements-actions.txt
-rasa==2.8.0 # NB! when updating, make sure to also update:
+rasa==2.8.1 # NB! when updating, make sure to also update:
             # * rasa-sdk base image in Dockerfile
             # * format of stories, domain.yml etc. if necessary
```

Comment:
See comment I made. I forgot to click on "add your review" first.

Reply:
Now I cannot see the comment here, I meant to say to add here that the rasa version in the root docker-compose file also needs to be updated.